Core/EntityManager/src/test/java/eu/neclab/ngsildbroker/entityhandler/services/EntityServiceTest.java

```diff
@@ -208,117 +240,102 @@ public void createMessageThrowsAlreadyExistTest() throws ResponseException, Exce
 	}
 
 	/**
-	 * this method is use for update the entity
+	 * this method is use for update the entity if Attribute is not same
 	 */
 	@Test
-	public void updateMessageTest() throws Exception {
+	public void updateMessageAttrIsNotExistTest() throws Exception {
 		try {
-			MockitoAnnotations.initMocks(this);
 			ArrayListMultimap<String, String> entityIds = ArrayListMultimap.create();
 			entityIds.put(AppConstants.INTERNAL_NULL_KEY, "urn:ngsi-ld:Vehicle:A103");
-			//when(entityInfoDAO.getAllIds()).thenReturn(entityIds);
-			Method postConstruct = EntityService.class.getDeclaredMethod("loadStoredEntitiesDetails");
-			postConstruct.setAccessible(true);
-			postConstruct.invoke(entityService);
 			ReflectionTestUtils.setField(entityService, "directDB", true);
 			UpdateResult updateResult = new UpdateResult();
 			Mockito.doReturn(entityPayload).when(entityInfoDAO).getEntity(any(), any());
 			multimaparr.put("content-type", "application/json");
 			Gson gson = new Gson();
-			Map<String, Object> resolved = gson.fromJson(updatePayload, Map.class);
+			Map<String, Object> resolved = gson.fromJson(updatePayloadforwrongattr, Map.class);
 			Map<String, Object> entityBody = (Map<String, Object>) JsonUtils.fromString(entityPayload);
 			UpdateEntityRequest request = new UpdateEntityRequest(multimaparr, "urn:ngsi-ld:Vehicle:A103", entityBody,
 					resolved, null);
 			updateResult = entityService.updateEntry(multimaparr, "urn:ngsi-ld:Vehicle:A103", resolved);
-			Assert.assertEquals(request.getUpdateResult().getUpdated(), updateResult.getUpdated());
-			verify(entityService).updateEntry(any(), any(), any());
+			Assert.assertEquals(request.getUpdateResult().getNotUpdated(), updateResult.getNotUpdated());
```

Comment:
[Minor] please check this assert equals condition  I think we need to compare with message..

Reply:
It throws array list object so we not compare with message